webui/src/pages/setup/index.jsx

```diff
@@ -54,48 +54,57 @@ server:
   endpoint_url: ${window.location.protocol}//${window.location.host}${API_ENDPOINT}
 `);
         return (
+            <>
             <Row>
                 <Col md={{offset: 2, span: 8}}>
                     <Card className="setup-widget">
-                        <Card.Header>Congratulations</Card.Header>
-                        <Card.Body>
+                        <Card.Body className={"after-setup-card"}>
+                            <h2>You&apos;re all set!</h2>
                             <Card.Text>
-                                Database was initialized and admin user was created.<br/>
-                                Here are your credentials:<br/>
+                                Here are the credentials for your first user:<br/>
                             </Card.Text>
-                            <hr/>
-                            <Table borderless hover>
-                                <tbody>
-                                <tr>
-                                    <td>Key ID</td>
-                                    <td><code>{setupData.access_key_id}</code> <ClipboardButton variant="link" text={setupData.access_key_id} tooltip="Copy key ID"/></td>
-                                </tr>
-                                <tr>
-                                    <td>Secret Key</td>
-                                    <td><code>{setupData.secret_access_key}</code> <ClipboardButton variant="link" text={setupData.secret_access_key} tooltip="Copy secret key"/></td>
-                                </tr>
-                                </tbody>
-                            </Table>
-                            <Card.Text>
-                                Download the initial client configuration under your <code>$HOME/.lakectl.yaml</code> and keep a copy of the data for your login into the system<br/>
-                            </Card.Text>
-                            <Alert variant="warning">
+                            <div className={"ml-2 row mt-4"}>
+                                <div className={"col-3"}>Access Key ID:</div>
+                                <div className={"col-7"}><code>{setupData.access_key_id}</code> &#160;&#160;<ClipboardButton className={"copy-button"} variant="outline-dark" text={setupData.access_key_id} tooltip="Copy"/></div>
```

Comment:
Why not push the `variant="outline-dark"` to the css if it's shared between the 2 usages?

Reply:
The variant is a react attribute, you can't apply it using CSS